app/src/editor/canvas/components/ModuleSearch.jsx

```diff
@@ -268,15 +268,7 @@ export class ModuleSearch extends React.PureComponent<Props, State> {
             }
         })
 
-        return modules.sort(this.compareModules)
-    }
-
-    // Used for sorting module list. Sorts first by path and then by name.
-    compareModules = (a: any, b: any) => {
-        if (a.path === b.path) {
-            return a.name.localeCompare(b.name)
-        }
-        return a.path ? a.path.localeCompare(b.path) : 0
+        return modules
```

Comment:
I guess we still need to handle sorting at least in browse mode? Removing this will result in weird order of modules and categories.

Reply:
Good point, will update. Though I'll put that sorting step into the state so it only has to be done once.

Reply:
See 53bb40b